README.md

```diff
@@ -276,6 +304,10 @@ Compose.prototype.map = function(f) {
 
 #### `sequence` method
 
+```hs
+sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
```

Comment:
In Haskell `b -> f b` would be inferred from `f` rather than provided as an argument, so `f` would need to satisfy Applicative.

In JavaScript `f` doesn't need to satisfy Applicative, strictly speaking, though it must be capable of supporting Applicative.

`Apply f =>` is technically correct; `Applicative f =>` is more descriptive. The question is whether we should favour correctness or descriptiveness. Does anyone have a strong view on this?